examples/nodefeaturerule.yaml

```diff
@@ -5,12 +5,10 @@ metadata:
 spec:
   rules:
     - name: "my sample rule"
-      labels:
-        "my-sample-feature": "true"
+      extendedResources:
+        vendor.io/dynamic: "@kernel.version.major"
+        vendor.io/static: "123"
```

Comment:
Please don't remove the example that shows how to create labels. You can append a new rule demonstrating extended resources or alternatively extend the existing rule adding the extendedResources field there. The latter could be better because in principle you don't need to match the feature (in the matchFeatures`) in order to use its value in an extended resource (of course with that you can make sure that it exists)